resources/views/quizquestions/create.blade.php

```diff
@@ -13,29 +13,29 @@
                         </div>
                     </div>
                     <div class="card-body">
-                        <form method="POST" action="create" id="boolean">
+                        <form method="POST" action="{{route('quizzes.quizquestions.store', $quiz)}}" id="boolean">
```

Comment:
```suggestion
                        <form method="POST" action="{{ route('quizzes.quizquestions.store', $quiz) }}" id="boolean">
```
convention, thêm dấu cách vào nhé, a quang có cmt rùi đó, chỗ khác sửa tương tự